assets/stylesheets/parent_group_overview.scss

```diff
@@ -0,0 +1,12 @@
+.parent_group_overview_grouping_active {
+  font-weight: bold;
+  color: #666666;
```

Comment:
That's actually our default font color. Is there any special meaning to that color? Use either `$default-font-color` from `openqa_theme.scss` or extend that file.

Reply:
Thanks!